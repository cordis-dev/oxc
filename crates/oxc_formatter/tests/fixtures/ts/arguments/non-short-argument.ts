setTimeout(
  () => {
    //
  },
  timeout * Math.pow(1)
);

setTimeout(
  () => {
    //
  },
  true || isFinite(Infinity)
);

setTimeout(
  () => {
    //
  },
  call(f) as any
);

setTimeout(
  () => {
    //
  },
  call(f) satisfies any
);
