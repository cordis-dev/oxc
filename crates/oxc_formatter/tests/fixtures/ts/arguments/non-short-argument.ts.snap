---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
setTimeout(
  () => {
    //
  },
  timeout * Math.pow(1)
);

setTimeout(
  () => {
    //
  },
  true || isFinite(Infinity)
);

setTimeout(
  () => {
    //
  },
  call(f) as any
);

setTimeout(
  () => {
    //
  },
  call(f) satisfies any
);

==================== Output ====================
setTimeout(
  () => {
    //
  },
  timeout * Math.pow(1),
);

setTimeout(
  () => {
    //
  },
  true || isFinite(Infinity),
);

setTimeout(
  () => {
    //
  },
  call(f) as any,
);

setTimeout(
  () => {
    //
  },
  call(f) satisfies any,
);

===================== End =====================
